cpp/oneapi/dal/algo/linear_kernel/common.cpp

```diff
@@ -20,28 +20,28 @@ namespace oneapi::dal::linear_kernel {
 
 class detail::descriptor_impl : public base {
 public:
-    double k = 1.0;
-    double b = 0.0;
+    double scale = 1.0;
+    double shift = 0.0;
 };
 
 using detail::descriptor_impl;
 
 descriptor_base::descriptor_base() : impl_(new descriptor_impl{}) {}
 
-double descriptor_base::get_k() const {
-    return impl_->k;
+double descriptor_base::get_scale() const {
+    return impl_->scale;
 }
 
-double descriptor_base::get_b() const {
-    return impl_->b;
+double descriptor_base::get_shift() const {
+    return impl_->shift;
 }
 
-void descriptor_base::set_k_impl(const double value) {
-    impl_->k = value;
+void descriptor_base::set_scale_impl(const double value) {
```

Comment:
Can we move it to protected?

Reply:
It solves only part of the problem - `descriptor_base` is still in public interfaces. Do we assume that it can be used by customers? I was thinking that it is a service functionality and is a part of our internal architecture